source/20_spec/config_spec_en.rst

```diff
@@ -0,0 +1,139 @@
+DI-engine Config Specification
+===============================
+
+The config submitted by the developer needs to comply with the following specifications to ensure the ease of use, readability, and extensibility of the config.
+
+The config of DI-engine includes two parts: main_config and create_config.
+
+example link
+-------------
+
+Example of a common algorithm (DQN):
+
+https://github.com/opendilab/DI-engine/blob/main/dizoo/atari/config/serial/pong/pong_dqn_config.py
+
+Example of an algorithm (SQIL) containing a model or data:
```

Comment:
May be Soft Q Imitation Learning as SQIL does not belong to the most popular set of algorithms.